truenas_api_client/__init__.py

```diff
@@ -700,14 +715,39 @@ def _jobs_callback(self, mtype: str, *, fields: JobFields, **message):
                         event = job['__ready'] = Event()
                     event.set()
 
-    def _jobs_subscribe(self):
+    def _jobs_subscribe(self, silent=False):
         """Subscribe to job updates, calling `_jobs_callback` on every new event."""
         self._jobs_watching = True
-        self.subscribe('core.get_jobs', self._jobs_callback, sync=True)
+        try:
+            self.subscribe('core.get_jobs', self._jobs_callback, sync=True, _log_py_enoauthenticated=not silent)
+        except Exception as e:
+            self._jobs_watching = False
+
+            if silent and self._is_enoauthenticated(e):
+                return
 
-    def call(self, method: str, *params, background=False, callback: _JobCallback | None=None,
-             job: Literal['RETURN'] | bool=False, register_call: bool | None=None,
-             timeout: float | UndefinedType=undefined) -> Any:
+            raise
+
+    def _is_enoauthenticated(self, e: BaseException):
```

Comment:
Should this be `_is_enotauthenticated` (instead of `_is_enoauthenticated`) in all 3 locations?